src/feed-list.scss

```diff
@@ -0,0 +1,127 @@
+@import "./mixins";
+
+$block: #{$fd-namespace}-feed-list;
+
+.#{$block} {
+  $fd-feed-list-padding-start: 1rem;
+  $fd-feed-list-padding-end: 0.5rem;
+  $fd-feed-list-margin-start: 0.5rem;
+  $fd-feed-list-margin-end: 0;
+  $fd-feed-list-icon-color: var(--sapContent_IconColor);
+
+  @include fd-reset();
+  @include fd-flex();
+
+  background-color: var(--sapField_Background);
+  max-width: 40rem;
+
+  .#{$block}__thumb {
+    @include fd-reset();
+
+    display: inline-flex;
+    margin-left: $fd-feed-list-margin-end;
+    margin-right: $fd-feed-list-margin-start;
+
+    @include fd-rtl() {
+      margin-left: $fd-feed-list-margin-start;
+      margin-right: $fd-feed-list-margin-end;
+    }
+  }
+
+  &__body {
+    @include fd-reset();
+
+    display: flex;
+    padding: 1rem $fd-feed-list-padding-end 1rem $fd-feed-list-padding-start;
+
+    & + .#{$block}__body {
+      border-top: var(--sapList_BorderWidth) solid var(--sapList_BorderColor);
+    }
+  }
+
+  &__actions {
+    @include fd-reset();
+
+    height: 2rem;
+  }
+
+  &__icon {
+    @include fd-reset();
+
+    @include fd-icon-overwrite() {
+      color: $fd-feed-list-icon-color;
+    }
+  }
+
+  .#{$block}__title {
+    @include fd-reset();
+
+    margin-left: $fd-feed-list-margin-start;
+    margin-right: $fd-feed-list-margin-end;
+
+    @include fd-rtl() {
+      margin-left: $fd-feed-list-margin-end;
+      margin-right: $fd-feed-list-margin-start;
+    }
+
+    &--icon {
+      color: $fd-feed-list-icon-color;
+    }
+  }

```

Comment:
You don't need to set any specified styles for this element. Use action sheet for actions in examples